### DISCUSSION TITLE
Using VSCode extension with antlr4-maven-plugin

### DISCUSSION THREAD

**Discussion by raffian** [mike-lischke/vscode-antlr4#223] [Q&A]
Simple questions:

1. Does this extension require the antlr4 maven plugin? I don't believe that's true but just want to confirm.
2. Are there known issues if using the extension and plugin in the same project?

**Comment by mike-lischke** [comment]
All you need for this extension is VS Code and Java. It doesn't use Maven nor have I ever heard of problems when you have a Maven plugin/extension installed.
